controller/work_item_link_type.go

```diff
@@ -89,8 +91,7 @@ func enrichLinkTypeList(ctx *workItemLinkContext, list *app.WorkItemLinkTypeList
 	for categoryID := range categoryIDMap {
 		modelCategory, err := ctx.Application.WorkItemLinkCategories().Load(ctx.Context, categoryID)
 		if err != nil {
-			jerrors, httpStatusCode := jsonapi.ErrorToJSONAPIErrors(err)
-			return ctx.ResponseData.Service.Send(ctx.Context, httpStatusCode, jerrors)
+			err
```

Comment:
Yes,

You might then change the code in line 173 of workitemtype.go controller to:
```
err = enrichLinkTypeList(linkCtx, appLinkTypes)
if err != nil {
	return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal("Failed to enrich link types: %s", err.Error()))
}
```

and in line 211:
```
			err = enrichLinkTypeList(linkCtx, appLinkTypes)
			if err != nil {
				return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal("Failed to enrich link types: %s", err.Error()))
			}
```